1-js/03-code-quality/03-comments/article.md

```diff
@@ -1,38 +1,38 @@
-# Comments
+# टिप्पणियाँ
 
-As we know from the chapter <info:structure>, comments can be single-line: starting with `//` and multiline: `/* ... */`.
+जैसा कि हम अध्याय जानकारी:संरचना से जानते हैं, टिप्पणियाँ सिंगल-लाइन हो सकती हैं: // से शुरू, और मल्टीलाइन : /* ... */ से शुरू । 
 
-We normally use them to describe how and why the code works.
+हम आमतौर पर उनका उपयोग यह वर्णन करने के लिए करते हैं कि कोड कैसे और क्यों काम करता है।
 
-At first sight, commenting might be obvious, but novices in programming often use them wrongly.
+पहली नजर में, टिप्पणी करना स्पष्ट हो सकता है, लेकिन प्रोग्रामिंग में नौसिखिए अक्सर उनका गलत इस्तेमाल करते हैं।
 
-## Bad comments
+## खराब टिप्पणियाँ
 
-Novices tend to use comments to explain "what is going on in the code". Like this:
+नौसिखिए "कोड में क्या हो रहा है" समझाने के लिए टिप्पणियों का उपयोग करते हैं। इस कदर:
 
 ```js
-// This code will do this thing (...) and that thing (...)
-// ...and who knows what else...
-very;
-complex;
-code;
+// यह कोड यह काम करेगा (...) और वह काम (...)
+// ... और कौन जानता है ...
+बहुत;
+जटिल;
+कोड;
```

Comment:
Inside the code blocks only comments can be translated. Please don't translate these.